Description: Use system Crypto++ library
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2011-05-17
--- a/src/makefile.unix
+++ b/src/makefile.unix
@@ -12,6 +12,8 @@
 
 DEFS=-DNOPCH -DUSE_SSL
 
+CRYPTOPPLIBS := $(shell pkg-config --libs libcrypto++)
+
 # for boost 1.37, add -mt to the boost libraries
 LIBS= \
  -Wl,-Bstatic \
@@ -30,6 +32,7 @@
 
 LIBS+= \
  -Wl,-Bdynamic \
+   $(CRYPTOPPLIBS) \
    -l gthread-2.0 \
    -l z \
    -l dl \
@@ -74,9 +77,7 @@
     obj/rpc.o \
     obj/script.o \
     obj/util.o \
-    obj/wallet.o \
-    cryptopp/obj/sha.o \
-    cryptopp/obj/cpu.o
+    obj/wallet.o
 
 
 all: bitcoin
@@ -85,9 +86,6 @@
 obj/%.o: %.cpp $(HEADERS)
 	$(CXX) -c $(CXXFLAGS) $(WXINCLUDEPATHS) -DGUI -o $@ $<
 
-cryptopp/obj/%.o: cryptopp/%.cpp
-	$(CXX) -c $(CXXFLAGS) -O3 -o $@ $<
-
 bitcoin: $(OBJS) obj/ui.o obj/uibase.o
 	$(CXX) $(CXXFLAGS) -o $@ $^ $(WXLIBS) $(LIBS)
 
@@ -109,5 +107,4 @@
 	-rm -f obj/*.o
 	-rm -f obj/nogui/*.o
 	-rm -f obj/test/*.o
-	-rm -f cryptopp/obj/*.o
 	-rm -f headers.h.gch
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -6,7 +6,7 @@
 #include "db.h"
 #include "net.h"
 #include "init.h"
-#include "cryptopp/sha.h"
+#include <cryptopp/sha.h>
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
 
