Description: Use system Crypto++ library
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2011-05-17
--- a/src/makefile.unix
+++ b/src/makefile.unix
@@ -12,6 +12,8 @@
 
 DEFS=-DNOPCH -DFOURWAYSSE2 -DUSE_SSL
 
+CRYPTOPPLIBS := $(shell pkg-config --libs libcrypto++)
+
 # for boost 1.37, add -mt to the boost libraries
 LIBS= \
  -Wl,-Bstatic \
@@ -30,6 +32,7 @@
 
 LIBS+= \
  -Wl,-Bdynamic \
+   $(CRYPTOPPLIBS) \
    -l gthread-2.0 \
    -l z \
    -l dl
@@ -48,9 +51,7 @@
     obj/irc.o \
     obj/main.o \
     obj/rpc.o \
-    obj/init.o \
-    cryptopp/obj/sha.o \
-    cryptopp/obj/cpu.o
+    obj/init.o
 
 
 all: bitcoin
@@ -59,9 +60,6 @@
 obj/%.o: %.cpp $(HEADERS)
 	$(CXX) -c $(CXXFLAGS) $(WXINCLUDEPATHS) -DGUI -o $@ $<
 
-cryptopp/obj/%.o: cryptopp/%.cpp
-	$(CXX) -c $(CXXFLAGS) -O3 -o $@ $<
-
 obj/sha256.o: sha256.cpp
 	$(CXX) -c $(CXXFLAGS) -msse2 -O3 -march=amdfam10 -o $@ $<
 
@@ -79,7 +77,6 @@
 clean:
 	-rm -f obj/*.o
 	-rm -f obj/nogui/*.o
-	-rm -f cryptopp/obj/*.o
 	-rm -f headers.h.gch
 	-rm -f bitcoin
 	-rm -f bitcoind
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -3,7 +3,7 @@
 // file license.txt or http://www.opensource.org/licenses/mit-license.php.
 
 #include "headers.h"
-#include "cryptopp/sha.h"
+#include <cryptopp/sha.h>
 
 
 
