Description: Avoid static linking - just do simple linking
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2011-05-17
--- a/src/makefile.unix
+++ b/src/makefile.unix
@@ -16,7 +16,6 @@
 
 # for boost 1.37, add -mt to the boost libraries
 LIBS= \
- -Wl,-Bstatic \
    -l boost_system \
    -l boost_filesystem \
    -l boost_program_options \
@@ -31,7 +30,6 @@
 endif
 
 LIBS+= \
- -Wl,-Bdynamic \
    $(CRYPTOPPLIBS) \
    -l gthread-2.0 \
    -l z \
