Description: Avoid arch-specific optimizations to miner code
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2010-12-28
--- a/src/makefile.unix
+++ b/src/makefile.unix
@@ -54,7 +54,7 @@
 
 
 obj/sha256.o: sha256.cpp
-	g++ -c $(CFLAGS) -msse2 -O3 -march=amdfam10 -o $@ $<
+	g++ -c $(CFLAGS) -o -O3 $@ $<
 
 bitcoin: $(OBJS) obj/ui.o obj/uibase.o obj/sha256.o
 	g++ $(CFLAGS) -o $@ $^ $(WXLIBS) $(LIBS)
